plugins/background/README.md

```diff
@@ -0,0 +1,32 @@
+# background
+
+This plugin adds [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background) utilities to Tailwind. Individual background properties are available by default, but occasionally providing the entire shorthand value can be useful, like for complex gradients, image-based patterns, or consolidating many properties.
+
+## Usage
+
+```js
+theme: {
+  background: {
+    glow:
+      'radial-gradient(58.13% 57.41% at 49.33% 78.4%, rgba(208, 2, 27, 0.6) 0%, rgba(208, 2, 27, 0) 100%)',
```

Comment:
Maybe we should use a shorter example for radial gradient, just so it can format on one line.